src/wdio/element-finder.ts

```diff
@@ -357,12 +364,13 @@ export class ElementFinder {
       if (AutomationEnvironment.isLocalMode()) {
         const element: WebdriverIO.Element = await this.findElement();
         if (AutomationEnvironment.getBrowserType() === BrowserType.Chrome) {
-          await AutomationEnvironment.getWorkingBrowser().action('pointer', { parameters: { pointerType: 'touch' } })
-            .move({ origin: element, x: 0, y: 0 })
-            .down({ button: 0 })
-            .pause(duration)
-            .up({ button: 0 })
-            .perform();
+          await AutomationEnvironment.getWorkingBrowser().execute(`
+            const touchStart = new TouchEvent('touchstart', { bubbles: true });
+            const touchEnd = new TouchEvent('touchend', { bubbles: true });
+            const el = arguments[0];
+            el.dispatchEvent(touchStart);
+            setTimeout(() => el.dispatchEvent(touchEnd), ${duration});
```

Comment:
This code change does not match with what this PR describes.
Moreover, this setTimeout should be avoided to ensure there's enough waiting time between touch start and end.
I would suggest to revert this changes and tackle them in another PR specific for long press.